astro.config.ts

```diff
@@ -47,6 +53,7 @@ export default defineConfig({
 	integrations: [
 		preact({ compat: true }),
 		sitemap({
+			filter: (page) => sitemapBlocklist.has(new URL(page).pathname),
```

Comment:
Would help if I got the logic the right way round 😅 

```suggestion
			filter: (page) => !sitemapBlocklist.has(new URL(page).pathname),
```